packages/playground/wordpress/src/index.ts

```diff
@@ -48,6 +50,11 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
     `
 	);
 
+	await php.writeFile(
+		'/internal/shared/mu-plugins/1-auto-login.php',
+		autoLoginMuPlugin
```

Comment:
Is this mu-plugin intended for use as broader platform feature? If not, could this mu-plugin be maintained and written as part of the `login` Blueprint step?

Reply:
> Is this mu-plugin intended for use as broader platform feature? I

Yes, it seems like the only way of supporting auto login in CLI, Studio, etc.

Reply:
> > Is this mu-plugin intended for use as broader platform feature? I
> 
> Yes, it seems like the only way of supporting auto login in CLI, Studio, etc.

Ah, thanks for explaining!